workflows/service_deploy.go

```diff
@@ -209,8 +211,8 @@ func (workflow *serviceWorkflow) serviceApplyCommonParams(namespace string, serv
 			params["HostListenerRulePriority"] = strconv.Itoa(workflow.priority + 1)
 		} else if svcStack != nil {
 			// no value in config, and this is an update...use prior value
-			params["PathListenerRulePriority"] = ""
-			params["HostListenerRulePriority"] = ""
+			params["PathListenerRulePriority"] = "1"
```

Comment:
This should be `""`...otherwise you'll always force priority to 1 on updates.

Reply:
I changed it to "1" because I was getting an error saying that this parameter had to be a Number.